### PR TITLE
Initialize swerve and vision

### PR DESCRIPTION
**Pull request by vichik123**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -1,26 +1,77 @@
...
                 exampleSubsystemIO = new ExampleSubsystemIOSim();
+                for (int i = 0; i < moduleIOs.length; i++) {
+                    moduleIOs[i] = new ModuleIOSim();
+                }
+                gyroIO = new GyroIOSim();
+                frontLeftVisionModule =
+                        new VisionModule(
+                                new VisionSimIO(
+                                        new PhotonCamera("Front left camera"),
+                                        new Transform3d(0.293, 0.293, 0.2, new Rotation3d()),
```
Comment (katzuv) [inline src/main/java/frc/robot/RobotContainer.java]: https://github.com/Galaxia5987/robot-2024/blob/49e1d00b7ee4bd137827cabb989bd0b2945a987f/src/main/java/frc/robot/RobotContainer.java#L64-L65
Duplicate, also should be in a different constant file (why not do that in another file)?

**Review by katzuv — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (vichik123) [reply src/main/java/frc/robot/RobotContainer.java]: 4b493526b5b891e1af2dd6cf7c25df441ef73c52

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -1,26 +1,107 @@
 package frc.robot;
```
Comment (katzuv) [inline src/main/java/frc/robot/RobotContainer.java:1]: This file will be long, but I'm not sure I have a good alternative. I'd at least put the vision init and swerve init code in different functions.

**Review by katzuv — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (vichik123) [reply src/main/java/frc/robot/RobotContainer.java:1]: e9883f326f94f9855e29c096bf204860a2349355

**Review by katzuv — APPROVED** [review]

**Pull request merged**